src/renderer/features/network/NetworksList/ui/EmptyNetworks.tsx

```diff
@@ -0,0 +1,24 @@
+import { useUnit } from 'effector-react';
+
+import { Icon, BodyText } from '@shared/ui';
+import { activeNetworksModel } from '../model/active-networks-model';
+import { inactiveNetworksModel } from '../model/inactive-networks-model';
+import { useI18n } from '@app/providers';
+
+export const EmptyNetworks = () => {
+  const { t } = useI18n();
+
+  const [activeNetworks, inactiveNetworks] = useUnit([
```

Comment:
this is what makes me think they should actually be grouped, we do group them in our head, but they come from 2 different models. Anyways, it's not a constructive comment, I'm just trying to prove my point.

Reply:
They are grouped in a single feature - `NetworkList`
Explicit division inside feature is more flexible and maintainable
One day `activeNetworkModel`/`inactiveNetworkModel` might grow and lead to the big mess if they were aggregated in a single file/model